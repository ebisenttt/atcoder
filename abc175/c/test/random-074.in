443978091581370 236741970254149 526504389708558
