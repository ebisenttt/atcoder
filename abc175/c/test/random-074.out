82526298127188
